java/src/main/java/ai/rapids/cudf/Aggregate.java

```diff
@@ -43,7 +43,7 @@ static Aggregate min(int index) {
   }
 
   static Aggregate avg(int index) {
```

Comment:
Should this remain `avg`?  This name was changed in the Table API, so wondering if it should be changed here for consistency.